omod/src/main/webapp/fragments/dashboardwidgets/discreteValues.gsp

```diff
@@ -0,0 +1,23 @@
+
+<div class="info-section">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label) }</h3>
+        <a href="${ ui.urlBind("/" + contextPath + app.url, [ "patient.uuid": patient.id ]) }" class="right">
+            <i class="icon-share-alt edit-action" title="Edit"></i>
+        </a>
+    </div>
+    <div class="info-body">
+        <% fields.each { concept, obs -> %>
+            <div>
+                <span style="font-family: 'OpenSansBold'";>${ ui.message(obs.label) }:</span>
+                <% if ( obs.minValue && obs.obsValue && obs.minValue.isNumber() && obs.obsValue.isNumber() &&
```

Comment:
We should add `minValue` as a number the first place, and validate that it's a number in the Controller.  By the time we  get here, minValue should be assumed to either be null or a number.

Reply:
ok